### PR TITLE
tox: remove cruft from file

### PR DESCRIPTION
**Pull request by andrewbonney**

### CODE REVIEW

**Comment by andrewbonney** [comment]
@simonrankine requesting a review on the same changes (roughly) for lots of repos

File: Jenkinsfile [hunk 1]
```diff
@@ -15,7 +15,7 @@
 
 pipeline {
     agent {
-        label "16.04&&ipstudio-deps"
+        label "ubuntu&&apmm-slave"
```
Comment (alexrawcliffe) [inline Jenkinsfile]: Whole line can be safely removed.

File: tox.ini [hunk 2]
```diff
@@ -29,44 +29,4 @@ commands =
...
-[testenv:py27-test]
-commands = python -m unittest discover -s tests
-deps = mock
-install_command = pip install --process-dependency-links {opts} {packages}
-
-[testenv:py3-test]
-commands = python -m unittest discover -s tests
-deps = mock
-install_command = pip install --process-dependency-links {opts} {packages}
+install_command = pip install {opts} {packages}
```
Comment (alexrawcliffe) [inline tox.ini]: Whole line can be safely removed.

File: tox.ini [hunk 3]
```diff
@@ -19,7 +19,7 @@ commands =
 deps =
     coverage
     mock
-install_command = pip install --process-dependency-links {opts} {packages}
+install_command = pip install {opts} {packages}
```
Comment (alexrawcliffe) [inline tox.ini]: Whole line can be safely removed.

**Pull request merged**